web/context.go

```diff
@@ -627,3 +627,14 @@ func (c *Context) RequireBotUserId() *Context {
 	}
 	return c
 }
+
+func (c *Context) RequirePrivacy() *Context {
```

Comment:
Given that this parameter is only related to a team, maybe is better to call it `RequireTeamPrivacy`

Reply:
I removed this now that the privacy field is moved into the body.